business/services.go

```diff
@@ -216,10 +228,19 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	conf := config.Get()
 	apiSpecFromAnnotation := svc.ObjectMeta.Annotations[conf.ApiDocumentation.Annotations.ApiSpecAnnotationName]
 	apiTypeFromAnnotation := svc.ObjectMeta.Annotations[conf.ApiDocumentation.Annotations.ApiTypeAnnotationName]
+	specURI, err2 := url.Parse(apiSpecFromAnnotation)
```

Comment:
Looks like you are not checking if the apidoc annotations exist. Because of this, a `/` path is being returned when no annotations are present, leading to showing the API Doc tab in the UI (which will show a "Unable to load the Spec" legent).

Please, add a check to for the apidoc annotations presence.